nexum_core/run_benchmarks.sh

```
@@ -1,136 +1,136 @@
-echo "📊 Results Summary:"
-echo "   - Logs: $RESULTS_DIR/*.log"
-echo "   - HTML Reports: $RESULTS_DIR/criterion/report/index.html"
-echo "   - Summary: $RESULTS_DIR/summary.md"
-echo ""
-echo "💡 To view HTML reports:"
-echo "   cd $RESULTS_DIR && python -m http.server 8000"
-echo "   Then open: http://localhost:8000/criterion/report/index.html"
-echo ""
-echo "🔍 To compare with previous runs:"
-echo "   ls benchmark_results/"
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)